client/src/main/webjar/organization-form/organization-form.html

```diff
@@ -59,6 +68,9 @@
             <input name="organizationFacebook"
                    ng-model="organizationForm.organization.facebookUrl"
                    type="url">
+            <div ng-messages="organizationForm.orgForm.organizationFacebook.$error">
+                <div ng-message="url">Organization Facebook page must be a valid url</div>
```

Comment:
Yea, I started to do that with zip code (12345, 12345-1234, or 12345 1234), but I was concerned about the real estate -- too wordy.